nginx.conf

```diff
@@ -5,6 +5,7 @@ server {
 
   location /inbox {
     alias /usr/share/nginx/html/;
+    try_files $uri /index.html;
```

Comment:
should we add it to other spa ? reason ?

Reply:
It is used because nginx will respond 404 to any page loaded that is not inbox (except for resources of course)
Eg: Loading /inbox/unifiedinbox/inbox would 404.  
I think this rule can be improved (to tell some urls patterns to try `/index.html` and other to try `$uri`)  
This should be added on other spas. It is not needed on devServer mode with webpack, because webpack handles this case.